core/templates/dev/head/pages/exploration_editor/feedback_tab/FeedbackTab.js

```diff
@@ -40,6 +40,7 @@ oppia.controller('FeedbackTab', [
 
     $scope.activeThread = null;
     $scope.userIsLoggedIn = null;
+    $scope.isThreadUpdated = false;
```

Comment:
This shouldn't start with a verb, since it's not a method/function. $scope.threadIsUpdated

Reply:
Done